### PR TITLE
Project calculator design

### PR DESCRIPTION
**Pull request by leonard33**
Created the front design of the calculator project

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import Calculator from './components/Calculator';
+import './App.css';
+
+function App() {
+  return (
+    <div className="App">
+      <Calculator />
+    </div>
+  );
+}
```
Comment (prolajumokeoni) [inline src/App.js:10]: - [ ] Your  App.js component should be class-based, please update this component to use class as it is part of the project requirements 👍🏿

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
+/* eslint-disable react/button-has-type */
```
Comment (prolajumokeoni) [inline src/components/Calculator.js:1]: - [ ] Kindly avoid disabling linters manually, please remove this line as there should be no custom change to the linters 👍🏿

File: src/components/Calculator.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+import './Calculate.css';
+
+const createDigit = () => {
+  const digits = [];
+  for (let i = 9; i >= 1; i -= 1) {
+    digits.push(<button key={i}>{i}</button>);
+  }
+  return digits;
+};
+// eslint-disable-next-line react/prefer-stateless-function
```
Comment (prolajumokeoni) [inline src/components/Calculator.js:12]: - [ ] Kindly avoid disabling linters manually, please remove this line as there should be no custom change to the linters 👍🏿

File: src/components/Calculator.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+            <div className="operations-top">
+              <button>AC</button>
+              <button>+/-</button>
+              <button>%</button>
+            </div>
+            <div className="digits">
+              {createDigit()}
+              <button>0</button>
+              <button>.</button>
+            </div>
```
Comment (prolajumokeoni) [inline src/components/Calculator.js:29]: - [ ] Good job working on this component 👏🏿  please add the border between each button so your design looks exactly like the original design 👍🏿 

Original Design          |  Your design     
:-------------------------:|:-------------------------:
calculator  | Screenshot_213

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @leonard33, 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design
✔️  Good PR title and Description



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by leonard33** [comment]
Correction review
- Use of class based components in app.js
- Removed custom change to the linters and disabling
- Border added between each button on the design

File: src/App.js [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
```
Comment (KDlamini) [inline src/App.js:29]: [OPTIONAL] kindly avoid leaving unused code as comments. It is best practice to follow the YAGNI principle, it's best to remove code that you do not need to make your codebase cleaner and readable.

**Review by KDlamini — APPROVED** [review]
### Hi @leonard33 🙋‍♀️,  

Excellent work! 💯 
You have done a great job implementing all the required changes. 🎊 🥳 
Your feature is complete! There is nothing else to say other than... it's time to merge it. :shipit: 
Congratulations!!!

# Status: Approved ✔️ 

simon

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Kindly look through the comments below 👇 for optional changes**
 
Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**